doc/source/user_guide/model-mapping.rst

```diff
@@ -11,8 +11,19 @@ purposes:
    exclusion list for model native regions, since only explicitly mentioned regions are
    selected. Any region not mentioned in *native_regions* will cause an error unless explicitly named in the *exclude_regions* section. This to avoid accidentally forgetting a region.
 2. Allowing for renaming of model native regions.
-3. Defining how model native regions should be aggregated to common
-   regions.
+3. Defining how model native regions should be aggregated to common regions.
+
+.. _native-vs-common-region:
+
+Difference between model-native and common regions
+---------------------------------------------------
+
+Model native regions are regions that are reported *directly* from an integrated
+assessment model. 
+
+Common regions, on the other hand, are calculated by *aggregating model native*
```

Comment:
Aggregating regions from one model or different models? Maybe this could be added. 

Also, maybe an example would be useful?

Reply:
For comparison you aggregate only regions from one model otherwise you'd be mixing results. I'll try to make it more clear. 
I've thought about adding an example but left it out as I wanted the explanation to be as short as possible. Seems like I lost clarity on the way. I'll add an example to make things more clear.